src/main/java/frc/robot/RobotPreferences.java

```diff
@@ -175,8 +175,14 @@ public static final class prefPitch {
     public static final SN_DoublePreference pitchP = new SN_DoublePreference("pitchP", 70);
     public static final SN_DoublePreference pitchI = new SN_DoublePreference("pitchI", 0);
     public static final SN_DoublePreference pitchD = new SN_DoublePreference("pitchD", 0);
+    public static final SN_DoublePreference pitchSupplyCurrentThreshold = new SN_DoublePreference(
+        "pitchSupplyCurrentThreshold", 0);
+    public static final SN_DoublePreference pitchWithSupplyTimeThreshold = new SN_DoublePreference(
```

Comment:
Can you make this name shorter? Maybe:
pitchSupplyTimeTreshold

Reply:
nuh uh